training/training.module

```diff
@@ -442,26 +750,120 @@ function theme_test_form_theme_function($form) {
 }
 
 /**
+ * Creates array of users id from db.
+ *
+ * Select from table text_message users id who created text message.
+ *
+ * @param string $table
+ *   Table name.
+ * @param string $alias
+ *   Table alias.
+ *
+ * @return array $table
+ *   Returns array of users id.
+ */
+function _training_select_uid_from_db($table, $alias) {
+  $uid = array();
+  $query = db_select($table, $alias);
+  $query
+    ->fields($alias, array('uid'))
+    ->distinct();
+  $result = $query->execute()->fetchAll();
```

Comment:
i fixed this but i used the other way:
$uid = $query->execute()->fetchCol();
because fetchAllKeyed needs two fields as parameters (key, value),
but i select only one field 'uid' from tables